crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -861,6 +926,21 @@ mod tests {
         assert_eq!(circle.eccentricity(), 0., "incorrect circle eccentricity");
     }
 
+    #[test]
+    fn ellipse_perimeter() {
```

Comment:
Maybe we should use relative equality instead of strict float equality, considering it's a numerical algorithm.